trieste/bayesian_optimizer.py

```diff
@@ -492,39 +492,45 @@ def optimize(
 
                 summary_writer = get_tensorboard_writer()
                 if summary_writer:
-                    with summary_writer.as_default():
+                    with summary_writer.as_default(step=step):
                         for tag in datasets:
                             with tf.name_scope(f"{tag}.model"):
                                 models[tag].log()
-                            tf.summary.scalar(
-                                f"{tag}.observation.best_overall",
-                                np.min(datasets[tag].observations),
-                                step=step,
+                            tf.summary.histogram(
```

Comment:
histogram makes sense only for batches, no?

Reply:
I mean it still works for batch size one (though it doesn't provide any more information that the best_in_batch scalar plot).